spec/import/numismatic_csv_transform_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Transforming Nuministics CSV file' do
+  let(:indexer) do
+    Traject::Indexer.new('command_line.filename' => fixture_file_path,
+                         'exhibit_slug' => slug,
+                         'agg_provider' => 'American Numismatics Society',
+                         'inst_id' => 'ans').tap do |i|
+      i.load_config_file('lib/traject/numismatic_csv_config.rb')
+    end
+  end
+  let(:fixture_file_path) { File.join(fixture_path, 'csv/numismatic_islam_department.csv') }
+  let(:data) { File.open(fixture_file_path).read }
+  let(:exhibit) { create(:exhibit) }
+  let(:slug) { exhibit.slug }
+
+  before do
+    allow(CreateResourceJob).to receive(:perform_later)
+  end
+
+  it 'does the transform' do
+    indexer.process(data)
+    expect(CreateResourceJob).to have_received(:perform_later) do |_id, _two, json|
+      dlme = JSON.parse(json)
+      expect(dlme['agg_provider']).to eq 'American Numismatics Society'
+
+      # MET Museum
+      expect(dlme['id']).to eq 'ans_1975-93-862'
+
+      expect(dlme['cho_contributor']).to eq ['Zaffar b. Shabba|al-Qasim b. `Ubayd Allah']
```

Comment:
In `Authority`, the pipe delimiter indicates multiple values (see http://numismatics.org/search/id/1975.93.862.xml).